scripts/notebook-instance-monitor/notebookapi.py

```diff
@@ -0,0 +1,230 @@
+import requests
+from datetime import datetime
+import getopt, sys
+import urllib3
+import boto3
+import json
+import os
+# os.system('pip install GPUtil==1.4.0')
```

Comment:
Generally suggest to remove lines if not needed, rather than commenting them out.

Reply:
thanks, removed in [this commit](https://github.com/w601sxs/amazon-sagemaker-notebook-instance-lifecycle-config-samples/commit/a17edc1cf86b753ba36e14020e4b085f7ebb45a6)